rest/data/user.go

```diff
@@ -20,16 +20,23 @@ func UpdateSettings(dbUser *user.DBUser, settings user.UserSettings) error {
 			Message:    "expected a Slack username, but got a channel",
 		}
 	}
+
 	settings.SlackUsername = strings.TrimPrefix(settings.SlackUsername, "@")
 	settings.Notifications.PatchFinishID = dbUser.Settings.Notifications.PatchFinishID
 	settings.Notifications.SpawnHostOutcomeID = dbUser.Settings.Notifications.SpawnHostOutcomeID
 	settings.Notifications.SpawnHostExpirationID = dbUser.Settings.Notifications.SpawnHostExpirationID
 	settings.Notifications.CommitQueueID = dbUser.Settings.Notifications.CommitQueueID
 
+	slackTarget := fmt.Sprintf("@%s", settings.SlackUsername)
```

Comment:
can we move it under the if check so that we don't do this if settings.SlackMembeId != ""

Reply:
It's easier if I initialize it to one thing to start with, and since this is just a print and not a db check, I think it's okay. If I move it under the if, I'd need to have it in an else which feels less clean to me.